### ISSUE TITLE
On the reproductivity of Ref-NeRF

### ISSUE THREAD

**Issue by resurgo97** [kakaobrain/nerf-factory#2]
Hi, I really appreciate your work, well done!
I have a question about your implementation.

How much does your work reproduce the results of Ref-NeRF?
The paper reports 33.99 of average PSNR on Blender dataset.
On the other hand, as I calculate the average PSNR of 8 scenes you report in the webpage, I get 33.27, slight lower than the official result.
Did I miss something, or do you have ideas on why there is a difference?

Thanks!

**Comment by seungjooshin** [comment]
Our implementation follows the hyperparameter setting written in the original paper, except for those below. It might occur the performance difference.

| | Original | Ours |
|:---|:---:|:---:|
| Iteration | 250K | 1M |
| Batch size | 16,384 | 4,096 |
| Learning rate | 2e-5 &rarr; 2e-3 | 5e-6 &rarr; 5e-4 |
| Warm-up | 512 | 2,500 |
| Bottleneck noise | std=0.1 | X |

**Comment by seungjooshin** [comment]
Our implementation follows the hyperparameter setting written in the original paper, except for those below. It might occur the performance difference.

| | Original | Ours |
|:---|:---:|:---:|
| Iteration | 250K | 1M |
| Batch size | 16,384 | 4,096 |
| Learning rate | 2×10<sup>-3</sup> &rarr; 2×10<sup>-5</sup> | 5×10<sup>-4</sup> &rarr; 5×10<sup>-6</sup> |
| Warm-up | 512 | 2,500 |
| Bottleneck noise | std=0.1 | X |

+) On the coarse stage, we use 0.1 times the loss weights on the fine stage. The original paper uses same loss weights.
